rdr_service/resource/generators/workbench.py

```diff
@@ -210,6 +210,15 @@ def make_resource(self, src_pk_id, backup=False):
         :param backup: if True, get from backup database instead of Primary.
         :return: resource object
         """
+
+        def v_to_e(val, enum_=WorkbenchResearcherYesNoPreferNot):
```

Comment:
Can we rename this to encompass the action, maybe `value_to_enum`?

Reply:
Done